### PR TITLE
Fix for tests

### PR DESCRIPTION
**Pull request by remicollet**
* test suite start failing with justinrainbow/json-schema 1.4.4
* 1 test is failing on distro using "jsonc" (instead of non-free json ext.)

### CODE REVIEW

File: tests/Composer/Test/Json/JsonFileTest.php [hunk 1]
```diff
@@ -54,6 +54,9 @@ public function testParseErrorSkipsEscapedBackslash()
 
     public function testParseErrorDetectSingleQuotes()
     {
+        if (defined('JSON_PARSER_NOTSTRICT')) {
+            $this->markTestSkipped('jsonc issue');
```
Comment (staabm) [inline tests/Composer/Test/Json/JsonFileTest.php]: could we be more descriptive here? maybe even link to issues within the issue-tracker?

**Comment by remicollet** [comment]
Link to upstream issue added.

**Comment by stof** [comment]
you need to update the lock file too, because the locked version of json-schema (1.4.1) does not match the constraint anymore

**Comment by remicollet** [comment]
Sorry but I don't know what is the policy for the composer.lock file.
Its update pull a lot of changes...

**Comment by alcohol** [comment]
I hope you don't mind, I included your fixes in my PR, but limited the lock file update to json-schema only instead of all the dependencies.

**Comment by remicollet** [comment]
> I hope you don't mind

That's fine.

**Comment by alcohol** [comment]
@remicollet I think you can safely close this PR now. 
Your commits were merged together with my PR :-)

**Comment by remicollet** [comment]
Nice, thanks.

**Pull request closed without merging**